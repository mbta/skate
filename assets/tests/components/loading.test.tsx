import React from "react"
import renderer from "react-test-renderer"
import Loading from "../../src/components/loading"

test("renders loading text", () => {
  const tree = renderer.create(<Loading />).toJSON()

  expect(tree).toMatchSnapshot()
})
