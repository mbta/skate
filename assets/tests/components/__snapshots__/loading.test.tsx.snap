// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders loading text 1`] = `"loading..."`;
